### PR TITLE
New docs folder / Contributing, internal and code-of-conduct

### PR DESCRIPTION
**Pull request by siddwarkhedkar**
## Manual Testing Steps

1.

## Pre-Submission Checklist:

- [ ] You've updated the CHANGELOG following the steps [here](https://github.com/klaviyo/magento2-klaviyo#making-updates)
- [ ] **Internal Only** - If this is a release, please confirm the following:
  - [ ] The links in the changelog have been updated to point towards the new versions
  - [ ] The version has been incremented in the following places: module.xml and composer.json

**NOTE:** Please use the [Changelogger](https://pypi.org/project/changelogged/) cli tool to manage versioned file upgrades.

### CODE REVIEW

**Comment by klaviyoit** [comment]
https://klaviyo.tpondemand.com/entity/163130

File: docs/INTERNAL.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+3) Upon approval merge your changes into master.
+    1) If this is a version update:
+        1) Checkout the master branch locally, make sure to pull down any changes that were just merged.
+        2) Use `git log` to find the merge commit's checksum.
+        3) Tag this commit with the version you just incremented: `git tag -a {version} {commit hash}` or just use `git tag -a {version}`.
+        4) Push the tag to the remote repository: `git push origin 1.0.1` replacing with the version you've just tagged.
+
+## Releasing a new version
+1) Submit the new version for approval on the Magento Marketplace using partner account
+2) [Update Packagust / Composer](https://klaviyo.atlassian.net/wiki/spaces/EN/pages/1530593327/Contributing+to+and+Updating+the+Magento+2+Extension#Update-Packagist%2FComposer) using the linked internal documentation
```
Comment (smoucka) [inline docs/INTERNAL.md:34]:
```suggestion
2) [Update Packagist / Composer](https://klaviyo.atlassian.net/wiki/spaces/EN/pages/1530593327/Contributing+to+and+Updating+the+Magento+2+Extension#Update-Packagist%2FComposer) using the linked internal documentation
```

File: docs/INTERNAL.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        3) Move any unreleased changes into your version update under the appropriate categories.
+        4) Update the `[Unreleased]` link to point from your new version to HEAD e.g. if you're updating to version 1.0.2 you'd update the link from `1.0.1...HEAD` to `1.0.2...HEAD`.
+        5) Add a link to your new version. The tag won't yet exist but you can create a link to the tag you will create shortly. Follow the pattern of previous links.
+3) Upon approval merge your changes into master.
+    1) If this is a version update:
+        1) Checkout the master branch locally, make sure to pull down any changes that were just merged.
+        2) Use `git log` to find the merge commit's checksum.
+        3) Tag this commit with the version you just incremented: `git tag -a {version} {commit hash}` or just use `git tag -a {version}`.
+        4) Push the tag to the remote repository: `git push origin 1.0.1` replacing with the version you've just tagged.
```
Comment (smoucka) [inline docs/INTERNAL.md:30]: I think we could probably remove this whole section as it's covered in the IES Release strategy doc linked in the above section and handled entirely by our CD pipeline now

File: README.md [hunk 3]
```diff
@@ -86,27 +86,6 @@ Magento 2
...
-        3) Move any unreleased changes into your version update under the appropriate categories.
-        4) Update the `[Unreleased]` link to point from your new version to HEAD e.g. if you're updating to version 1.0.2 you'd update the link from `1.0.1...HEAD` to `1.0.2...HEAD`.
-        5) Add a link to your new version. The tag won't yet exist but you can create a link to the tag you will create shortly. Follow the pattern of previous links.
-3) Upon approval merge your changes into master.
-    1) If this is a version update:
-        1) Checkout the master branch locally, make sure to pull down any changes that were just merged.
-        2) Use `git log` to find the merge commit's checksum.
-        3) Tag this commit with the version you just incremented: `git tag -a {version} aeb8c682cebe7acee94506d3e4bfff2e5755e8c1` or just use `git tag -a {version}`.
-        4) Push the tag to the remote repository: `git push origin 1.0.1` replacing with the version you've just tagged.
-
```
Comment (smoucka) [inline README.md:109]: Can we add links to the contributing and code of conduct like we have for Presta?

**Review by smoucka — APPROVED** [review]

**Pull request merged**